source/2019-06-08-the-ember-times-issue-102.md

```diff
@@ -14,12 +14,12 @@ responsive: true
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [RFC for an alternative to Controllers 🥴](https://github.com/emberjs/rfcs/pull/499)
+There are **two** RFCs related to the possibility of eventually removing controllers from Ember.
+ - [RFC 499](https://github.com/emberjs/rfcs/pull/499) Proposes an alternative way of rendering templates for a particular route. (if some of you remember routable components, this isn't that 😉)
+ - [RFC 380](https://github.com/emberjs/rfcs/pull/380) Proposas a service for handling query parameters, as well as a `@queryParam` **decorator** for accessing query parameters anywhere in your app: Routes, Components, etc. 🎉
+ 
+With these two RFCs, we'll eventually be able to deprecate Controllers. 🔥🔥🔥
```

Comment:
Since the RFCs are, to my understanding, still in a comment period, I think a better way to conclude this article is to remove the current sentence and make a call-to-action (ask readers to provide comments for RFCs).

Maybe check out issues #95 and #98 for examples?